website/docs/cheatsheet/distributed-training.md

```diff
@@ -6,524 +6,335 @@ keywords:
+* The Azure ML environment contains Horovod and MPI. The PyTorch and TensorFlow curated GPU environments come pre-configured with Horovod and its dependencies.
+* Create an `MpiConfiguration` with your desired distribution.
 
-No matter which launch style user choose, users can follow these steps:
-1. Use an AzureML environment with the preferred deep learning framework and MPI. AzureML provides [curated environment](https://docs.microsoft.com/en-us/azure/machine-learning/resource-curated-environments) for popular frameworks.
-2. Define `MpiConfiguration` with the desired `process_per_node` and `node_count`. `process_per_node` should be equal to the number of GPUs per node for per-process-launch, or set to 1 for per-node-launch if user script will be responsible to launch processes for each node.
-3. Set the `mpi` section of the [runconfig](https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.runconfig.runconfiguration?view=azure-ml-py) using the `MpiConfiguration`. When using `ScriptRunConfig` simply pass it as the value for argument `distributed_job_config`.
-4. Prepare the user script and map environment variables AzureML set up to your needs. See [detailed instruction](#environment-variables-from-openmpi) and [examples](#examples).
+#### Example
+* [azureml-examples: TensorFlow distributed training using Horovod](https://github.com/Azure/azureml-examples/tree/main/workflows/train/tensorflow/mnist-distributed-horovod)
 
-Here is a code snippet:
-```python
-from azureml.core import Workspace, ScriptRunConfig, Environment
-from azureml.core.runconfig import MpiConfiguration
+### DeepSpeed
 
-curated_env_name = 'AzureML-PyTorch-1.6-GPU'
-pytorch_env = Environment.get(workspace=ws, name=curated_env_name)
-mpiconfig = MpiConfiguration(process_count_per_node=4, node_count=2)
-
-run_cf = ScriptRunConfig(
-    source_directory= 'src' ,
-    script="train.py",
-    compute_target=compute_target,
-    distributed_job_config=mpiconfig,
-    environment=pytorch_env 
-)
-
-# submit the runconfig and run the job
-experiment = Experiment(ws, "experiment_name")
-experiment.submit(run_cf)
-```
+To run distributed training with the [DeepSpeed](https://www.deepspeed.ai/) library on Azure ML, do not use DeepSpeed's custom launcher. Instead, configure an MPI job to launch the training job [with MPI](https://www.deepspeed.ai/getting-started/#mpi-and-azureml-compatibility).
 
-:::caution
-To use AzureML mpirun, the base docker image used by the run need to have Open MPI. They are included in [all AzureML default GPU base images](https://github.com/Azure/AzureML-Containers). User is responsible to 
-install one of these when using custom base image. AzureML also provides [curated environment](https://docs.microsoft.com/en-us/azure/machine-learning/resource-curated-environments) for popular frameworks. 
-:::
+Take care of the following:
+* The Azure ML environment contains DeepSpeed and its dependencies, Open MPI, and mpi4py.
```

Comment:
@aminsaied I meant the user should make sure their environment has those dependencies. I'll make the wording more clear. We have that curated environment but I don't know if we should advertise it here since I'm not sure if this was a one-off curated env - let me know what you think